### PR TITLE
Added a Sugarcane resource and a converter to change .apsimx files 

### PR DESCRIPTION
**Pull request by hol353**
Working on #6729

### CODE REVIEW

**Comment by hol353** [comment]
retest this please jenkins

**Review by hol430 — CHANGES_REQUESTED** [review]
Perhaps it's worth running this converter over the existing sugarcane files? This should reduce the file size by quite a bit by the looks of things.

File: ApsimNG/Presenters/MainPresenter.cs [hunk 1]
```diff
@@ -1253,7 +1253,7 @@ private void OnShowConverter(object sender, EventArgs args)
                 int version = Models.Core.ApsimFile.Converter.LatestVersion;
                 ClearStatusPanel();
                 string bin = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
-                string resources = Path.Combine(bin, "..", "Models", "Resources");
+                string resources = Path.Combine(bin, "..", "..", "..", "Models", "Resources");
```
Comment (hol430) [inline ApsimNG/Presenters/MainPresenter.cs:1256]: This is nasty, but not really related to this pull request - I'll create a separate issue for this.

**Review by hol430 — APPROVED** [review]
Thanks, looks good to me.

**Comment by hol430** [comment]
retest this please jenkins

**Pull request merged**